docker-compose.dev.yml

```diff
@@ -21,15 +21,15 @@ services:
       dockerfile: Dockerfile
     container_name: postgres_db
     ports:
-      -"5432:5432"
+      - "5432:5432"
```

Comment:
We can remove ports and still connect django because they are in the same network although we should probably make that explicit by adding the network here.

Reply:
Hey i added the new network check it please, its working fine with me !